src/appleseed.cli/houdinitilecallbacks.h

```diff
@@ -66,8 +66,8 @@ class HRmanPipeTileCallbackFactory
 {
   public:
     HRmanPipeTileCallbackFactory(
-        int                 socket_number,
-        bool                progressive_mode,
+        const int           socket_number,
```

Comment:
I only added the missing `const` qualifier, I didn't want to introduce too many changes. But your question is very relevant.